libs/community/langchain_community/tools/azure_ai_services/image_analysis.py

```diff
@@ -19,19 +19,31 @@ class AzureAiServicesImageAnalysisTool(BaseTool):
     """Tool that queries the Azure AI Services Image Analysis API.
 
     In order to set this up, follow instructions at:
-    https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/quickstarts-sdk/image-analysis-client-library-40
+    https://learn.microsoft.com/azure/ai-services/computer-vision/quickstarts-sdk/image-analysis-client-library-40
+
+    Attributes:
+    azure_ai_services_key (Optional[str]): The API key for Azure AI Services.
+    azure_ai_services_endpoint (Optional[str]): The endpoint URL for Azure AI Services.
+    visual_features Any: The visual features to analyze in the image, can be set as
+        either strings or VisualFeatures.
```

Comment:
It might be helpful to show where this type comes from

```suggestion
        either strings or ~azure.ai.vision.imageanalysis.models.VisualFeatures.
```